packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1885,6 +1896,114 @@ export class ProjectService extends BaseService {
         return results;
     }
 
+    async streamResultsToLocalFile(
```

Comment:
function that creates stores the download file in the database, creates a file stream and pass a function to `write`. 

We should create something similar for s3

Reply:
in the ~database~ temp folder